utils/open-vm-tools/Makefile

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=open-vm-tools
+PKG_VERSION:=10.3.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-10430147.tar.gz
+PKG_SOURCE_URL:=https://github.com/vmware/open-vm-tools/releases/download/stable-$(PKG_VERSION)
+PKG_HASH:=364cd0fdfa5a05e872d08609659e6231ec99788669f7ebba24bfb8c94168daef
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-10430147
```

Comment:
This line could be dropped.

Reply:
I put 10430147 in $PKG_REV.
Is this OK?
Is the intention different?

Reply:
I checked OpenWrt base system code.  There is no mention of `PKG_REV` in the `include/` directory and no other package Makefile uses it other than those of the build system itself inside `toolchain/` and `tools/`. 

I would suggest not using `PKG_REV`.  Though it's not documented so, vars with `PKG_` prefix should be reserved for use by the build system.

If `10430147` is something monotonically increasing as maintained by the upstream project, I think maybe it can be put in the `PKG_VERSION` var.  Otherwise just leave in the `PKG_SOURCE` should be safe and fine.

Reply:
I put it in PKG_SOURCE.
I wanted to use the PKG_VERSION in the download URL directory name.